### DISCUSSION TITLE
可以部署在 GitHub pages 吗？Q: || be deployed on GitHub pages? Q:

### DISCUSSION THREAD

**Discussion by ghost** [walinejs/waline#2012] [Q&A] [answered]
### Describe the bug | 问题描述

我托管在GitHub的网站想搞一个【https://zelihole.github.io/】

### Website URL | 问题网站

https://zelihole.github.io/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

GitHub

---
### Describe the bug | Problem description

I want to build a website hosted on GitHub【https://zelihole.github.io/】

### Website URL | Problem Website

https://zelihole.github.io/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

Can GitHub

**Comment by lizheming** [comment, marked as answer]
Waline 可以使用在基于 GitHub Pages 部署的博客中。但 Waline 本身需要服务端能力，无法部署在 GitHub Pages 静态服务上。

---
Waline can be used in blogs deployed on GitHub Pages. However, Waline itself requires server-side capabilities and cannot be deployed on GitHub Pages static services.
